src/Tribe/Utils/Color.php

```diff
@@ -513,4 +513,61 @@ private static function _checkHex( $hex ) {
 
 		return $color;
 	}
+
+	/**
+	 * Get color that contrasts the most.
+	 * 
+	 * @since TBD
+	 *
+	 * @param string $hex_color  Hex value of color to compare.
+	 * @param string $lite_color Hex value of dark contrast color.
+	 * @param string $dark_color Hex value of light contrast color.
```

Comment:
Are these right? I'm a bit confused.

Reply:
Yes. You pass in two other colors (light and dark) to compare the first color to. It returns the one that contrasts the most.

Reply:
@codingmusician but then `$lite_color` has the description `Hex value of dark contrast color` and `$dark_color` the other way around. Just double checking.

Reply:
Oh! Whoops. LOL